docs/source/requests.md

```diff
@@ -2953,7 +2955,64 @@ A new Agreement needs to be sent instead.
 }
 ```
 
+### LEDGERS_FREEZE
 
+Freeze deprecated ledgers (default ledgers such as the domain, config, pool, and audit ledgers cannot be frozen). If a ledger is frozen it can be used for reading but not for writing. It will not be caught up by new nodes and can be safely removed (frozen ledgers can't be unfreeze). After freeze you can delete frozen ledger via [remove_ledger.py] (https://github.com/hyperledger/indy-node/blob/master/scripts/remove_ledger.py), but reeding from the ledger will become impossible.
```

Comment:
```suggestion
Freeze deprecated ledgers (default ledgers such as the domain, config, pool, and audit ledgers cannot be frozen). If a ledger is frozen it can be used for reading but not for writing. It will not be caught up by new nodes and can be safely removed (frozen ledgers can't be unfrozen). After freeze you can delete frozen ledger via [remove_ledger.py] (https://github.com/hyperledger/indy-node/blob/master/scripts/remove_ledger.py), but reading from the ledger will become impossible.
```

Reply:
Thanks for catching this. I fixed the typos while making more substantial changes to this paragraph.